src/tests/transform-result.spec.ts

```diff
@@ -1134,3 +1168,197 @@ describe('transformComment', (): void => {
 
     processTestData(testData, transformComment)
 })
+
+describe('transformFunctionAlias', (): void => {
+    const testData: TestData<Variant.FunctionAlias, FunctionAliasEntry> = [
+        [
+            {
+                done: false,
+                incomplete: false,
+                value: null,
+                variant: Variant.FunctionAlias
+            },
+            {
+                hit: 0,
+                done: false,
+                index: 0,
+                name: '',
+                variant: Variant.FunctionAlias
+            }
+        ],
```

Comment:
`value` can (or rather _should_) never be `null`, so having a test for it seems unnecessary.